tests/storage-vm

```diff
@@ -317,17 +317,20 @@ for poolDriver in $poolDriverList; do
         lxc delete -f v1
         lxc storage unset "${poolName}" volume.size
 
-        if [ "${poolDriver}" = "lvm" ]; then
+        if [ "${poolDriver}" = "lvm" ] || [ "${poolDriver}" = "ceph" ] || [ "${poolDriver}" = "powerflex" ]; then
                 echo "==> Change volume.block.filesystem on pool and create VM"
                 lxc storage set "${poolName}" volume.block.filesystem xfs
                 lxc init "${TEST_IMG:-ubuntu-minimal-daily:22.04}" v1 --vm -s "${poolName}"
                 lxc start v1
                 waitInstanceReady v1
                 lxc info v1
 
-                echo "==> Checking VM config disk filesyste is XFS"
+                # Ensure the VM volume ignores volume.block.filesystem.
+                lxc storage volume get "${poolName}" virtual-machine/v1 block.filesystem | grep -q "ext4"
```

Comment:
```suggestion
                echo "==> Ensure the VM volume ignores volume.block.filesystem"
                volume_fs="$(lxc storage volume get "${poolName}" virtual-machine/v1 block.filesystem)"
                echo "${volume_fs}" | grep -F "ext4"
```
Maybe the above would help figure why it failed in https://github.com/canonical/lxd-ci/actions/runs/8012265928/job/21887207157?pr=80#step:6:1775
Removing `-q` as

Reply:
Hasn't been backported yet. https://github.com/canonical/lxd/pull/12950

Reply:
But good to add.

Reply:
Of course it's not yet in 5.0/edge, face palm.

In the suggested change, I also converted the comment into an echo as I find them useful, especially in long scripts like `tests/storage-vm`.

Reply:
> In the suggested change, I also converted the comment into an echo as I find them useful, especially in long scripts like `tests/storage-vm`.

Yeah I agree, I've been trying to do that with the microcloud tests as well.